stytch/consumer/totps.go

```diff
@@ -0,0 +1,131 @@
+package consumer
+
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/stytchauth/stytch-go/v8/stytch"
+	"github.com/stytchauth/stytch-go/v8/stytch/consumer/totps"
+	"github.com/stytchauth/stytch-go/v8/stytch/stytcherror"
+)
+
+type TOTPsClient struct {
+	C *stytch.Client
+}
+
+func NewTOTPsClient(c *stytch.Client) *TOTPsClient {
+	return &TOTPsClient{
+		C: c,
+	}
+}
+
```

Comment:
It looks like we're missing two functions that are wrappers for other endpoints - `AuthenticateWithClaims` and `RecoverWithClaims`